src/main.rs

```diff
@@ -62,31 +89,62 @@ async fn main() {
 
     let mut windows_count = 0;
     let total_windows = 100;
+    
+    #[cfg(unix)]
     let windows: &mut [Option<SpawnedWindow>] = &mut vec![None; total_windows];
-
-    windows[windows_count] = Some(
-        spawn_window(
-            "USERNAME",
-            "TEST",
-            &[],
-            positions[position_idx],
-            monitor_geometry,
-        )
-        .await,
-    );
+    #[cfg(windows)]
+    let windows: &mut [Option<WindowsWindow>] = &mut vec![None; total_windows];
+
+    #[cfg(unix)]
+    {
+        windows[windows_count] = Some(
+            spawn_window(
+                "USERNAME",
+                "TEST",
+                &[],
+                positions[position_idx],
+                monitor_geometry,
+            )
+            .await,
+        );
+    }
+    #[cfg(windows)]
+    {
+        windows[windows_count] = Some(
+            WindowsWindow::new(
+                "USERNAME",
+                "TEST",
+                &[],
+                positions[position_idx],
+            )
+        );
+    }
+    
     position_idx += 1;
     position_idx %= positions.len();
     windows_count += 1;
     windows_count %= total_windows;
 
-    client.join("apika_luca".to_owned()).unwrap();
+    client.join("mictia00".to_owned()).unwrap();
```

Comment:
The tests/default-value are made with my channel.

```suggestion
    client.join("apika_luca".to_owned()).unwrap();
```